smithy-linters/src/main/java/software/amazon/smithy/linters/ShouldHaveUsedTimestampValidator.java

```diff
@@ -127,6 +129,16 @@ public List<ValidationEvent> structureShape(StructureShape shape) {
             public List<ValidationEvent> unionShape(UnionShape shape) {
                 return validateUnion(shape, model, patterns);
             }
+
+            @Override
+            public List<ValidationEvent> timestampShape(TimestampShape shape) {
+                return Collections.emptyList();
```

Comment:
Timestamp shapes are not instances of string or number so they would be covered by existing default.

Reply:
Ah ok, I'll remove it